controllers/openflow/flows.go

```diff
@@ -38,14 +38,17 @@ const (
 	tableEntry           = 0
 	tableNetworkPolicies = 10
 	tableClassification  = 20
-	tableLocalARP        = 30
-	tableL3Forwarding    = 40
-	tableL2Forwarding    = 50
-	tableL3Rewrites      = 60
-	tableL2Rewrites      = 70
-	tableProxy           = 80
-	tableNAT             = 90
-	tableAudit           = 100
+	tableL3Forwarding    = 30
+	tableL2Forwarding    = 40
+	tableL3Rewrites      = 50
+	tableL2Rewrites      = 60
+	tableProxy           = 70
+	tableNAT             = 80
+	tableAudit           = 90
+	tableLocalARP        = 100
+
+	// all pods have a fixed mac addr
+	podMacAddr = "aa:bb:cc:dd:ee:ff"
```

Comment:
this is defined twice (here and veth.go), would a common home for it make sense?

Reply:
A single package/file for 1 const seemed overkill but agree we should revisit this soon